tests/test_integration.py

```diff
@@ -1,26 +1,20 @@
 from unittest import mock
 from unittest.mock import patch, Mock
-import pytest
+import pytest, mongomock
 from dotenv import load_dotenv, find_dotenv
 import os
 
 from todo_app.app import create_app
 
 @pytest.fixture
 def client():
-    # Use our test integration config instead of the 'real' version
-    try:
-        file_path = find_dotenv('.env.test')
-        load_dotenv(file_path, override=True)
-    except:
-        pass
-    
-    # Create the new app.
-    test_app = create_app()
-    # Use the app to create a test_client that can be used in our tests.
-    with test_app.test_client() as client:
-        yield client
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
 
+    with mongomock.patch(servers=(('fakemongo.com', 27017),)):
+        test_app = create_app()
+        with test_app.test_client() as client:
+         yield client
 
 @patch('requests.request')
```

Comment:
Now that we're not relying on Trello, we should be able to clear out all of the Trello/requests specific stuff. We can then replace what's necessary - basically adding a fake task to our DB so that we can check it shows up in the test. So adding a function like you have in your session_items file that gets called at the start of your test:

```
def add_fake_items():
    newitem = {
        "status":"ToDo",
        "title": "My Complete Task"
    }

    mongo_url = os.getenv('MONGO_URL')
    mongo_db = os.getenv('MONGO_DB')

    client = pymongo.MongoClient(mongo_url)
    db = client[mongo_db]
    items = db.items
    items.insert_one(newitem)
```